2022/24xxx/CVE-2022-2421.json

```diff
@@ -0,0 +1,100 @@
+{
```

Comment:
Expected location (2022/2xxx/CVE-2022-2421.json) of file does not match actual location (2022/24xxx/CVE-2022-2421.json)